packages/ng/multi-select/displayer/counter-displayer/counter-displayer.component.ts

```diff
@@ -0,0 +1,97 @@
+import { AsyncPipe, NgFor, NgIf, NgPlural, NgPluralCase } from '@angular/common';
+import { ChangeDetectionStrategy, Component, DestroyRef, ElementRef, inject, Input, OnInit, ViewChild } from '@angular/core';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { FormsModule } from '@angular/forms';
+import { ILuOptionContext, LU_OPTION_CONTEXT, ɵLuOptionOutletDirective } from '@lucca-front/ng/core-select';
+import { InputDirective } from '@lucca-front/ng/form-field';
+import { LuTooltipModule } from '@lucca-front/ng/tooltip';
+import { switchMap } from 'rxjs/operators';
+import { BehaviorSubject, of } from 'rxjs';
+import { LuMultiSelectInputComponent } from '../../input';
+
+@Component({
+	selector: 'lu-multi-select-counter-displayer',
+	standalone: true,
+	imports: [AsyncPipe, LuTooltipModule, NgIf, NgFor, NgPlural, NgPluralCase, ɵLuOptionOutletDirective, FormsModule, InputDirective],
+	template: `
+		<div class="multipleSelect-displayer mod-filter" [class.is-filled]="(selectedOptions$ | async)?.length > 0">
+			<input
+				class="multipleSelect-displayer-search"
+				type="text"
+				[attr.aria-expanded]="select.isPanelOpen"
+				[attr.aria-activedescendant]="select.activeDescendant$ | async"
+				[attr.aria-controls]="ariaControls"
+				[disabled]="select.disabled$ | async"
+				[readonly]="!select.searchable"
+				#inputElement
+				ngModel
+				(ngModelChange)="select.clueChanged($event)"
+				[placeholder]="placeholder$ | async"
+				role="combobox"
+				aria-haspopup="listbox"
+				luInput
+			/>
+			<div class="multipleSelect-displayer-filter" *ngIf="selectedOptions$ | async as selectedOptions">
+				<div class="multipleSelect-displayer-chip chip mod-unkillable" *ngIf="selectedOptions?.length === 1">{{ selectedOptions[0]?.name }}</div>
```

Comment:
Options does not always have a name, we should use `displayerTemplate | optionTemplate` as in [default displayer](https://github.com/LuccaSA/lucca-front/blob/feat/multiselect-counter-displayer/packages/ng/multi-select/displayer/default-displayer.component.ts#L38)